src/Agent/NewRelic/Agent/Core/Configuration/ReportedConfiguration.cs

```diff
@@ -665,7 +665,7 @@ public ReportedConfiguration(IConfiguration configuration)
         [JsonProperty("agent.disable_file_system_watcher")]
         public bool DisableFileSystemWatcher => _configuration.DisableFileSystemWatcher;
 
-        [JsonProperty("agent.ai_monitoring.enabled")]
+        [JsonProperty("ai_monitoring.enabled")]
```

Comment:
We might want to include a comment that the name of this setting is important because it is used by the backend.

Reply:
Can do!